api/modules/feedback/views.py

```diff
@@ -30,3 +31,44 @@ def add_feedback(request):
 
     success_message = "Sucessfully added new feedback."
     return Response(success_message, status=status.HTTP_201_CREATED)
+
+
+@api_view(['GET'])
+def get_feedback_id(request, feedback_id):
```

Comment:
Add a check if the user is authorized to view the feedback or not.
Something similar to [this](https://github.com/project-travel-mate/server/blob/master/api/modules/trips/views.py#L52)